docs/testing.md

```diff
@@ -0,0 +1,30 @@
+# Testing
+
+## Unit testing
+
+### Quick 
+
+Ensure dependencies:
+- make
+- python 3.7 or higher (?)
+
+```bash
+python -m venv venv
+source vevn/bin/activate
```

Comment:
is it necessary to have a `venv` running now?

Reply:
nope!

Reply:
Excellent! Can you add something to this document so it's clear that you don't have to have the python stuff installed locally, and it can be run just using the docker image?

Reply:
Yes, the testing doc is rather skeletal, but the python manual testing instructions are gone.